### ISSUE TITLE
Improve color contrast & readability throughout 

### ISSUE THREAD

**Issue by moates** [moates/onlyBans#27]
- [ ] Text of “music-is-gonna-start-put-in-headphones.exe” has a contrast ratio of 2.15:1 (text too light)
- [ ] Sidebar text of “Only Bans” and “Goal...Wallet... Week” has a contrast
ratio of 2.74:1 (very low contrast between text and background)
- [ ] outing-prelude: avoid white background, check contrast
- [ ] suggested-friends: put a solid color background behind text
- [ ] “You get a DM from a sex worker you follow” needs to be on a solid color background, as does “Continue to your next week”
- [ ] attendWorkshop: Put all text on page in a solid color background
- [ ] permabanned-appeal: Make the solid background behind text more opaque
- [ ] start-week: Make solid color box behind text more opaque
- [ ] skip-verify-phone: Lower contrast between text and background
- [ ] skip-verify-more: Lower the contrast between text and background
- [ ] noReasonBan, all ban pages: for browser window graphic “flagged” text at top should be darker (contrast is too light)
- [ ]  noReason-real-story (probably all real-story pages): “IRL.exe” text at top should be darker (contrast is too light)
- [ ] outing-side-story, stolen-content: White text on blue background’s contrast is too low.
- [ ] close-account: Put a contrasting opaque color box behind the text.
- [ ] got-money-ending: Put a contrasting opaque-ish color box behind the text.
- [ ] ending: “The-end.exe” text is too light (too low contrast)
- [ ] About: no white bkg
